src/v2/view-builder/views/SuccessView.js

```diff
@@ -1,11 +1,32 @@
+import { _, loc } from 'okta';
+
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
 import Util from '../../../util/Util';
-import { loc } from 'okta';
 
 const Body = BaseForm.extend({
   title () {
-    return  loc('oie.success.redirect', 'login');
+    let titleString = loc('oie.success.text.signingIn', 'login');
+    // For more info on the API response available in appState, see IdxResponseBuilder.java
+    const app = this.options.appState.get('app');
+    const user = this.options.appState.get('user');
+
+    if (_.isEmpty(app)) {
+      return titleString;
+    }
+
+    const {label: appInstanceName, name: appDisplayName} = app;
+    const {identifier: userEmail} = user;
+
+    const appName = appInstanceName ? appInstanceName : appDisplayName;
+
+    if (appName && userEmail) {
+      titleString = loc('oie.success.text.signingIn.with.appName.and.identifier', 'login', [appName, userEmail]);
```

Comment:
- ` return loc(..)` 
- change `titleString` to be `const`

Reply:
I think separating all the title variants from a single variable is harder to read in the end. This function is currently just returning titleString, but if I return the loc functions without naming them and the first titleString, there'll be 3 return statements that look unrelated. But they're not unrelated, they're all just a titleString.